api/pow/client/client_test.go

```diff
@@ -0,0 +1,150 @@
+package client
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	tutil "github.com/textileio/go-threads/util"
+	ffsRpc "github.com/textileio/powergate/ffs/rpc"
+	"github.com/textileio/textile/api/apitest"
+	"github.com/textileio/textile/api/common"
+	hc "github.com/textileio/textile/api/hub/client"
+	"github.com/textileio/textile/core"
+	"github.com/textileio/textile/integrationtest/pg"
+	"github.com/textileio/textile/util"
+	"google.golang.org/grpc"
+)
+
+const powAddr = "127.0.0.1:5002"
+
+func TestPowClient(t *testing.T) {
```

Comment:
Simple integration test to make sure all the wires connect from the client all the way through to Powergate.